Makefile

```diff
@@ -141,6 +143,15 @@ index-rollover-integration-test: docker-images-elastic
 	go clean -testcache
 	bash -c "set -e; set -o pipefail; $(GOTEST) -tags index_rollover -coverpkg=./... -coverprofile cover-index-rollover.out $(STORAGE_PKGS) $(COLORIZE)"
 
+# Don't detect data race because testbed has race condition issue
```

Comment:
you mean the OTEL testbed? Can we fix it?

Reply:
yes, it's OTEL testbed. I've tried to fix it but got stuck. Apparently, in the `opentelemetry-collector-contrib` repository, the OTEL testbed run the tests without detecting data race which I overlooked before trying to use it. I also opened a discussion to about this (https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/30287) but still got no answer.

The main problem is that when we create a new testbed testcase with NewTestCase(), it starts a goroutine that log stats of the data sent (LoadGenerator) dan received (MockBackend) (https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/testbed/testbed/test_case.go#L126C18-L126C18) but later we must start a MockBackend which it write a startTime variable that is used by the log stats goroutine hence the data race occur.